src/main/resources/territories-list.json

```diff
@@ -0,0 +1 @@
tamp":"2016-12-15T12:15:07Z","key":"GS","item":[{"official-name":"South Georgia and the South Sandwich Islands","name":"South Georgia and South Sandwich Islands","territory":"GS"}]},"ES-ML":{"index-entry-number":"47","entry-number":"47","entry-timestamp":"2016-12-15T12:15:07Z","key":"ES-ML","item":[{"official-name":"Melilla","name":"Melilla","territory":"ES-ML"}]},"GU":{"index-entry-number":"34","entry-number":"34","entry-timestamp":"2016-12-15T12:15:07Z","key":"GU","item":[{"official-name":"Territory of Guam","name":"Guam","territory":"GU"}]},"AE-DU":{"index-entry-number":"24","entry-number":"24","entry-timestamp":"2016-12-15T12:15:07Z","key":"AE-DU","item":[{"official-name":"Emirate of Dubai","name":"Dubai","territory":"AE-DU"}]},"KY":{"index-entry-number":"17","entry-number":"17","entry-timestamp":"2016-12-15T12:15:07Z","key":"KY","item":[{"official-name":"Cayman Islands","name":"Cayman Islands","territory":"KY"}]},"TC":{"index-entry-number":"74","entry-number":"74","entry-timestamp":"2016-12-15T12:15:07Z","key":"TC","item":[{"official-name":"Turks and Caicos Islands","name":"Turks and Caicos Islands","territory":"TC"}]},"TF":{"index-entry-number":"29","entry-number":"29","entry-timestamp":"2016-12-15T12:15:07Z","key":"TF","item":[{"official-name":"French Southern Territories","name":"French Southern Territories","territory":"TF"}]},"CW":{"index-entry-number":"22","entry-number":"22","entry-timestamp":"2016-12-15T12:15:07Z","key":"CW","item":[{"official-name":"Country of Curaçao","name":"Curaçao","territory":"CW"}]},"CX":{"index-entry-number":"18","entry-number":"18","entry-timestamp":"2016-12-15T12:15:07Z","key":"CX","item":[{"official-name":"Territory of Christmas Island","name":"Christmas Island","territory":"CX"}]},"BAT":{"index-entry-number":"14","entry-number":"14","entry-timestamp":"2016-12-15T12:15:07Z","key":"BAT","item":[{"official-name":"British Antarctic Territory","name":"British Antarctic Territory","territory":"BAT"}]},"PF":{"index-entry-number":"28","entry-number":"28","entry-timestamp":"2016-12-15T12:15:07Z","key":"PF","item":[{"official-name":"French Polynesia","name":"French Polynesia","territory":"PF"}]},"TK":{"index-entry-number":"72","entry-number":"72","entry-timestamp":"2016-12-15T12:15:07Z","key":"TK","item":[{"official-name":"Tokelau","name":"Tokelau","territory":"TK"}]},"PM":{"index-entry-number":"65","entry-number":"65","entry-timestamp":"2016-12-15T12:15:07Z","key":"PM","item":[{"official-name":"Saint Pierre and Miquelon","name":"Saint Pierre and Miquelon","territory":"PM"}]},"PN":{"index-entry-number":"57","entry-number":"57","entry-timestamp":"2016-12-15T12:15:07Z","key":"PN","item":[{"official-name":"Pitcairn, Henderson, Ducie and Oeno Islands","name":"Pitcairn, Henderson, Ducie and Oeno Islands","territory":"PN"}]},"UM-71":{"index-entry-number":"48","entry-number":"48","entry-timestamp":"2016-12-15T12:15:07Z","key":"UM-71","item":[{"official-name":"Midway Islands","name":"Midway Islands","territory":"UM-71"}]}}
```

Comment:
Probably worth running these through a JSON pretty-printer. Everything being on one line will make it very difficult to review changes in future.

Reply:
we would not review these file but just replace them, they are big and not for humans.

Reply:
JSON files have been ran through a prettier - change implemented.